paramiko/ssh_exception.py

```diff
@@ -233,3 +233,13 @@ class ConfigParseError(SSHException):
     """
 
     pass
+
+
+class AuthenticationTimeoutException(SSHException):
```

Comment:
Should be a subclass of `AuthenticationException`.

Reply:
Fixed.